apps/central-scan/backend/src/fujitsu_scanner.ts

```diff
@@ -131,6 +135,7 @@ export class FujitsuScanner implements BatchScanner {
       `--batch=${join(directory, `${dateStamp()}-ballot-%04d.${this.format}`)}`,
       `--batch-print`,
       `--batch-prompt`,
+      `--bgcolor=black`,
```

Comment:
This is a good opportunity to start adding some comments to these arguments. Why is this flag important?